sqlitedict.py

```diff
@@ -303,7 +315,13 @@ def terminate(self):
 
     def __del__(self):
         # like close(), but assume globals are gone by now (do not log!)
-        self.close(do_log=False)
+        try:
+            self.close(do_log=False)
+        except Exception:
+            # prevent error log flood in case of multiple SqliteDicts
+            # closed after connection lost (exceptions are always ignored
+            # in __del__ method.
+            pass
```

Comment:
No, getting rid of the flood of logged exceptions is the entire reason for this PR.